server/src/test/java/com/cloud/network/lb/LoadBalancingRulesManagerImplTest.java

```diff
@@ -100,7 +100,5 @@ public void testGetLoadBalancerServiceProviderFail() {
         when(_networkMgr.getProvidersForServiceInNetwork(networkMock, Network.Service.Lb)).thenReturn(new ArrayList<>());
 
         Network.Provider provider = lbr.getLoadBalancerServiceProvider(loadBalancerMock);
-
-        Assert.assertNull(provider);
```

Comment:
why remove this assertion (and not replace)

Reply:
@DaanHoogland 
the test gives an exception so `provider` is always null.
(actually code smell asked me to remove it ...)